### PR TITLE
Add no pauschbetrag react page

### PR DESCRIPTION
**Pull request by frederike-ramin**
# Short Description
- Adds the page if person is not eligible for pauschbeträge

# Changes
- Add page and story and tests
- Only added one page that can be called by both steps: person A and B

# Feedback
- Do you see a problem with just adding one page?

# How to review this Pull Request
[Link to Confluence](https://digitalservice4germany.atlassian.net/wiki/spaces/STL/pages/117637157/Development+Process#Approve-or-comment) (internal)

### CODE REVIEW

File: webapp/client/src/stories/NoPauschbetragPage.stories.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+function Template(args) {
+  return <NoPauschbetragPage {...args} />;
+}
+
+export const Default = Template.bind({});
+Default.args = {
+  stepHeader: {
+    title:
+      "Leider haben Sie keinen Anspruch auf behinderungsbedingte Pauschbeträge.",
```
Comment (JulianRoesner) [inline webapp/client/src/stories/NoPauschbetragPage.stories.js:18]: We are still providing the title from the backend for consistency? Or is there another reason?

File: webapp/client/src/pages/NoPauschbetragPage.test.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+it("should render step header title", () => {
+  render(<NoPauschbetragPage {...props} />);
+  expect(screen.getByText("Title")).toBeInTheDocument();
+});
+
+it("should ignore step header intro", () => {
+  render(<NoPauschbetragPage {...props} />);
+  expect(screen.queryByText("Intro")).not.toBeInTheDocument();
+});
```
Comment (JulianRoesner) [inline webapp/client/src/pages/NoPauschbetragPage.test.js:22]: I know that we have those tests in other places, too. I am wondering if we really need those tests? We are also not testing if they are ignoring other props. Wouldn't it be better to tests if the correct intro is set?

File: webapp/client/src/lib/translations.js [hunk 3]
```diff
@@ -65,6 +65,12 @@ const translations = {
         label: "Merkzeichen aG",
       },
     },
+    noPauschbetrag: {
+      intro: {
+        p1: "Ein Pauschbetrag wird erst ab einem Grad der Behinderung von 20 gewährt. Liegt der Grad der Behinderung darunter, gibt es keinen Anspruch auf einen Pauschbetrag.",
+        p2: "<bold>Hinweis:</bold> Auch ohne Pauschbetrag können Sie behinderungsbedingte Aufwendungen absetzen. Und zwar im Bereich Krankheitskosten und weitere außergewöhnliche Belastungen.",
```
Comment (JulianRoesner) [inline webapp/client/src/lib/translations.js:71]: That is nice. I did not know that this is so easy.

**Review by JulianRoesner — APPROVED** [review]
LGTM! I think we are totally fine with one page in the frontend if we do not need different designs. Or is this the reason for setting the title in the backend? If so, I think maybe having two different pages would be preferable, but we can also do that when we move the title to the frontend.

File: webapp/client/src/stories/NoPauschbetragPage.stories.js (hunk 1 above)
Comment (frederike-ramin) [reply webapp/client/src/stories/NoPauschbetragPage.stories.js:18]: Ther eis a different title whether it's person A (single), Person A (joint taxes) or person B.

File: webapp/client/src/pages/NoPauschbetragPage.test.js (hunk 2 above)
Comment (frederike-ramin) [reply webapp/client/src/pages/NoPauschbetragPage.test.js:22]: Testing that the into is set is a bit senseless to me. But so might be testing this. I can totally delete this.

**Pull request merged**